src/coreclr/jit/emitarm64.cpp

```diff
@@ -10454,6 +10482,12 @@ void emitter::emitIns_R_R_R(instruction     ins,
             fmt = IF_SVE_CZ_4A;
             break;
 
+        case INS_sve_ld1b:
+        case INS_sve_ld1h:
+        case INS_sve_ld1w:
+        case INS_sve_ld1d:
+            return emitIns_R_R_R_I(ins, size, reg1, reg2, reg3, 0, opt);
```

Comment:
is there any reason why we can't call `emitIns_R_R_R_I()` directly from the caller?

Reply:
There's two ways to do this. Without any special coding in the table, it'll just automatically use the `R_R_R()` version because that's how many args there are in the intrinsic.

I see that elsewhere in NEON etc, this is how it's already done.

Alternatively, we could add an extra flag something like `HW_Flag_extra_zero_arg`  or just hardcode via the `HW_Flag_specialcodegen` route. That feels a lot of extra code.